.github/workflows/main.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+
+on:
+  push:
+    branches: [ master]
+  pull_request:
+    branches: [ master]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Create virtual environment
+        run: python -m venv venv
+
+      - name: Activate virtual environment
+        run: source venv/bin/activate
+
+      - name: Install dependencies
+        run: pip install -r requirements.txt && pip install -r tests/requirements.txt
```

Comment:
https://github.com/riptano/ccm/blob/master/.travis.yml#L3

Travis has different versions, should we use different ones as well?

Reply:
where do we run Travis? GitHub actions uses the newest I think

Reply:
we should converged, or remove, the travis stuff